app/src/main/java/org/jellyfin/androidtv/ui/browsing/StdBrowseFragment.java

```diff
@@ -1,17 +1,3 @@
-/*
- * Copyright (C) 2014 The Android Open Source Project
```

Comment:
I believe this license header needs to be kept. I believe that since the project as a whole is GPL v2 we cannot relicense this file from Apache v2.

Reply:
I wasn't sure about removing it either but I'd argue that pretty much none of the code in this file is from the Android OSS project now (except for the function names we inherit from the base class).

Reply:
This is the iniial version of the file:
https://github.com/jellyfin/jellyfin-androidtv/blob/96831e66be089b4fe2cd87cd6ed83f24a599caa9/MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/StdGridFragment.java

And I'm quite sure it is based on this file:
https://github.com/googlearchive/leanback-showcase/blob/master/app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/grid/GridExampleFragment.java

Reply:
Given that the effective license of this project is GPLv3 due to the inclusion of licensed code that is incompatible with GPLv2 prior to our fork, and since the GPLv3 is compatible with the Apache 2 license in that Apache 2 code can be relicensed as GPLv3, I believe this removal is fine.